great_expectations/data_context/store/json_site_store.py

```diff
@@ -24,13 +23,13 @@ def __init__(
             store_backend_module_name = store_backend.get(
                 "module_name", "great_expectations.data_context.store"
             )
-            store_backend_class_name = store_backend.get(
-                "class_name", "InMemoryStoreBackend"
-            )
+            # store_backend_class_name = store_backend.get(
+            #     "class_name", "InMemoryStoreBackend"
+            # )
             verify_dynamic_loading_support(module_name=store_backend_module_name)
-            store_backend_class = load_class(
-                store_backend_class_name, store_backend_module_name
-            )
+            # store_backend_class = load_class(
+            #     store_backend_class_name, store_backend_module_name
+            # )
```

Comment:
these objects are not being used

Reply:
let's instead keep them in but assign them to `_`? And add a comment?

Reply:
Done